third_party/FP16/LICENSE

```diff
@@ -0,0 +1,11 @@
+The MIT License (MIT)
```

Comment:
This will be the first non-Apache2-licensed code (that is not testing code), so it might be worth a mention in our toplevel LICENSE file.

Any chance there's an Apache2-licensed alternative?

Reply:
We could also reach out to Marat and ask if he's willing and able to dual-license it.

Reply:
A quick search didn't find any apache license libraries, only another MIT one. IANAL, but I was under the impression MIT is fine to include in apache.

Reply:
Yeah, I don't think it's a blocker to use this code, it is compatible AFAIU. But it is a different license so it seems worth mentioning at the top level.